packages/driver/src/cypress/error_messages.ts

```diff
@@ -134,23 +134,27 @@ export default {
   attachFile: {
     invalid_action: {
       message: `${cmd('attachFile')} \`action\` can only be \`input\` or \`drag-n-drop\`. You passed: \`{{action}}\`.`,
-      docsUrl: 'https://on.cypress.io/attachFile',
+      docsUrl: 'https://on.cypress.io/attachfile',
```

Comment:
fyi, you can also declare the `docsUrl` higher up on this tree and have it be the default for the errors, see `net_stubbing.docsUrl` for an example of this inheritance

Reply:
Ah, interesting. We should consider doing this for the whole file at some point - I didn't notice that was an option because most sections don't use it.